wdfn-server/waterdata/tests/test_views.py

```diff
@@ -67,6 +75,8 @@ def test_everything_okay(self, r_mock):
         )
         self.assertEqual(json_ld_response.status_code, 200)
         self.assertIsInstance(json.loads(json_ld_response.data), dict)
+        # see if call to Cooperator Lookup Service returns expected response
+        self.assertEqual(m_resp_lookup.status_code, 200)
```

Comment:
This is unnecessary and is not actually testing code in the view.  Your asserts should be looking at the response object. The only thing you might be able to do is check the rendered template to see if the sifta data made it there. That's overkill and would be brittle so just eliminate lines 78 and 79.